readme.md

```diff
@@ -36,6 +36,8 @@
 
 ## Introduction
 
+[![Greenkeeper badge](https://badges.greenkeeper.io/monicahq/monica.svg)](https://greenkeeper.io/)
```

Comment:
@asbiin can you put the badge more in the top of this page? We already have two badges there, we should group them.

Reply:
Yes of course, the badge has been added by greekeeper itself, I'll do it.